### PR TITLE
More benchmarking papers.

### PR DESCRIPTION
**Pull request by ltratt**

### CODE REVIEW

File: softdev.bib [hunk 1]
```diff
@@ -2445,3 +2445,37 @@ @inproceedings{oliveira14howmuch
     year = 2014,
     month = Feb
 }
+
+@article{mcilroy19spectre,
+    author = "Ross McIlroy and Jaroslav Sevc{\'{\i}}k and Tobias Tebbi and Ben L. Titzer and Toon Verwaest",
```
Comment (vext01) [inline softdev.bib]: Did you check this non-ascii glyph renders ok?

**Comment by vext01** [comment]
One small question.

File: softdev.bib (hunk 1 above)
Comment (ltratt) [reply softdev.bib]: It does render OK, but I've used a simplification that also renders the same way (so I think is better) -- see what you think.

Comment (vext01) [reply softdev.bib]: Much more readable :)

**Comment by vext01** [comment]
Please squash.

**Comment by ltratt** [comment]
Squashed.

**Pull request merged**